src/wase/ecs/entity.cpp

```diff
@@ -3,6 +3,7 @@
 #include "components/transform.h"
 
 #include <string>
+#include <algorithm>
```

Comment:
This doesn't add anything, we can remove this.

Reply:
This is about MinGW support. This should be in file. Because std:finder part of algorithm. Otherwise it gives error.